Source/OxyPlot/Annotations/PathAnnotation.cs

```diff
@@ -386,11 +386,14 @@ public override void Render(IRenderContext rc, PlotModel model)
         /// </returns>
         protected override HitTestResult HitTestOverride(HitTestArguments args)
         {
-            var nearestPoint = ScreenPointHelper.FindNearestPointOnPolyline(args.Point, this.screenPoints);
-            double dist = (args.Point - nearestPoint).Length;
-            if (dist < args.Tolerance)
+            if (this.screenPoints != null)
```

Comment:
invert `if` statement to reduce nesting (Resharper recommends this, and I agree)